app/controllers/authentications_controller.rb

```diff
@@ -0,0 +1,25 @@
+class AuthenticationsController < ApplicationController
+
+  include RequireRecentSignin
```

Comment:
Just include in `ApplicationController`?  See the argument for just including it here too since not used that much.